src/library/components/SelectOption.tsx

```diff
@@ -0,0 +1,25 @@
+import { SelectHTMLAttributes, useEffect, useState } from 'react';
+
+interface Props extends SelectHTMLAttributes<HTMLSelectElement> {
+  label: string;
+  value: string;
+}
+
+let optionId = 0;
+
+const SelectOption = ({ label, value, ...selectAttrs }: Props): JSX.Element => {
+  const [optionIdValue, setOptionIdValue] = useState('');
+
+  useEffect(() => {
+    const idNum = ++optionId;
+    setOptionIdValue('Option' + idNum);
+  }, []);
+
+  return (
+    <option id={optionIdValue} value={value}>
+      {value}
```

Comment:
we may want to split this apart and have this be text and follow the spec for defaulting to text/label/value